docs/.vuepress/theme/components/FrameworksDropdown.vue

```diff
@@ -51,9 +51,16 @@ export default {
       return this.$page.currentFramework !== void 0;
     },
     imageUrl() {
+      const currentVersion = this.$page.currentVersion;
       const frameworkWithoutNumber = this.$page.currentFramework.replace(/\d+$/, '');
+      let src = `/docs/${currentVersion}/${this.$page.currentFramework}${this.$page.frameworkSuffix}` +
+        `/img/pages/introduction/${frameworkWithoutNumber}.svg`;
 
-      return `/docs/${this.$page.currentVersion}/img/pages/introduction/${frameworkWithoutNumber}.svg`;
+      if (currentVersion === this.$page.latestVersion) {
```

Comment:
The logic of "if the current version is the latest version, modify the url" turns up in multiple places now, maybe it'd be a good idea to do a helper that does that?

Reply:
I'm not big fan of this idea, because helper needs 3 variables and not so much is calculated. However, I've introduced it within: https://github.com/handsontable/handsontable/pull/9482/commits/c9250054f743634b39bf3eb4641484eacc353a03.

Note: Please keep mind that I can't place function in `docs/.vuepress/helpers.js`, because that is module used by Node.js.